### PR TITLE
Integration tests for amp-video-iframe

### PR DESCRIPTION
**Pull request by alanorozco**
(in progress)

### CODE REVIEW

File: extensions/amp-video-iframe/0.1/amp-video-iframe.js [hunk 1]
```diff
@@ -120,13 +120,18 @@ class AmpVideoIframe extends AMP.BaseElement {
...
-        '<amp-video-iframe> does not allow ad iframes. ',
-        'Please use amp-ad instead.');
-
-    this.user().assert(!looksLikeTrackingIframe(element),
-        '<amp-video-iframe> does not allow tracking iframes. ',
-        'Please use amp-analytics instead.');
+    // TODO(alanorozco): On integration tests, `getLayoutBox` will returned
+    // a cached default value, which makes these assertions fail. Move to
+    // `describes.integration` to see if that fixes it.
+    if (!element.hasAttribute('i-amphtml-integration-test')) {
```
Comment (aghassemi) [inline extensions/amp-video-iframe/0.1/amp-video-iframe.js]: Would `!getMode().test` work instead?

Comment (alanorozco) [reply extensions/amp-video-iframe/0.1/amp-video-iframe.js]: That would be the case, but this is unit-tested as well, so we need to discern between the integration bug and the unit tests.

Comment (aghassemi) [reply extensions/amp-video-iframe/0.1/amp-video-iframe.js]: I see

**Comment by rsimha** [comment]
Seems like this has broken master:

https://travis-ci.org/ampproject/amphtml/jobs/455224522#L714

**Pull request merged**